examples/stable_diffusion/README.md

```diff
@@ -213,30 +213,3 @@ Then generate data:
 To generate one image:
 
 `python stable_diffusion.py --model_id stabilityai/sd-turbo --provider cpu --format qdq --guidance_scale 0 --seed 0 --num_inference_steps 1 --prompt "A baby is laying down with a teddy bear"`
-
-#### Evaluation result
```

Comment:
remove this in case the evaluation results are also derived works  https://stability.ai/license

Reply:
it is possible. for example, this also not evaluate them https://github.com/tgxs002/HPSv2?tab=readme-ov-file#custom-evaluation